cli/command/streams.go

```diff
@@ -1,23 +0,0 @@
-package command
-
-import (
-	"github.com/docker/cli/cli/streams"
-)
-
-// InStream is an input stream used by the DockerCli to read user input
-// Deprecated: Use github.com/docker/cli/cli/streams.In instead
-type InStream = streams.In
```

Comment:
Cory pointed out that the deprecation in this package used the wrong format (missing a newline). While most editors/IDEs may pick this up, Golang itself _doesn't_, which means that `pkg.go.dev` doesn't show it as deprecated (and Linters may not be warning about it); https://pkg.go.dev/github.com/docker/cli@v24.0.0-beta.1+incompatible/cli/command#InStream

<img width="1235" alt="Screenshot 2023-04-01 at 17 46 10" src="https://user-images.githubusercontent.com/1804568/229299664-71393ed8-4dc5-410c-9260-75a5052de9ef.png">

I had a quick look if I could find external consumers of this, and only found 2 (both were using a _very_ old version of docker (1.4.0), so likely no concern, but "just in case";

- I'll remove this commit from this PR
- Open a PR to fix the comments (and cherry-pick that into the 23.0 branch)
- Create a new draft PR to remove this (but for the 25.0 release, and we can merge that once we created the 24.0 branch)

Reply:
- ^^ removed that commit, and opened https://github.com/docker/cli/pull/4152